api/db-cdr/changelog/db.changelog-11.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+  <changeSet author="biopete" id="changelog-11.0"  runAlways="true" >
+    <validCheckSum>ANY</validCheckSum>
+    <preConditions onFail="MARK_RAN">
+      <tableExists tableName="ACHILLES_analysis" schemaName="cdr"></tableExists>
+    </preConditions>
+    <dropTable tableName="ACHILLES_analysis"/>
```

Comment:
can the table names all be lower case to be consistent? for example: achilles_analysis

Reply:
I don't want to change the ACHILLES_* table names and risk breaking atlas or achilles when it populates them. I don't like it either though.

Reply:
BigQuery is (FWIW) case sensitive about table names:

https://cloud.google.com/bigquery/docs/reference/standard-sql/lexical#case_sensitivity

@mark-velez I'm assuming in OHDSI code you're keeping achilles tables uppercase?

Reply:
Seems that our cloud mysql instance is also case sensitive:
mysql> select * from cdr.CRITERIA where type = 'DEMO';
ERROR 1146 (42S02): Table 'cdr.CRITERIA' doesn't exist

Reply:
I believe that is something we could override:

https://stackoverflow.com/questions/28507756/google-cloud-sql-disable-table-name-case-sensitivity

But since we can't in BQ, we should probably just stick to assuming case-sensitivity everywhere.